src/components/Datatable/Pagination/NavButton.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { IconButton } from '../../IconButton';
+import { IconButtonProps } from '../../IconButton/IconButton.types';
+
+const NavButton = styled(IconButton)
```

Comment:
It doesn't make sense to inherit `IconButton` component when you replace basically every style of that, just use `.button`. Then you don't need the `!important` we shouldn't be used at all.

Reply:
I am basically using it so I don't repeat code passing down iconName prop

Reply:
You don't need the iconName from IconButton prop at all, just use `Icon` component and add prop `direction` with values `prev` and `next` and decide which icon to show.